lib/pubsub/subscription.js

```diff
@@ -293,11 +293,17 @@ function Subscription(pubsub, options) {
    * //-
    * // Get the IAM policy for your subscription.
    * //-
-   * subscription.iam.getPolicy(function(err, policy) {
-   *   console.log(policy);
-   * });
    */
-  this.iam = new IAM(pubsub, this.name);
+  var iam;
+  Object.defineProperty(this, 'iam', {
+    get() {
```

Comment:
This throws an error for me running tests against `v0.12.x`. Should we use the `get: function() {` syntax instead? :/